src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,57 @@
+.stopwatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  position: relative;
+  background-image: url(../../images/watch.png);
+  background-color: $stopwatch-background-color;
+  top: 15vh;
+  border: 3px solid $stopwatch-border-color;
+  box-shadow: 0 0 120px $main-color;
+  transition: box-shadow 1s;
+  @include circle($stopwatch-size);
+  
+  &:hover {
+    box-shadow: 0 0 120px $hover-color;
+  }
+
+  &__center {
+    position: absolute;
+    background-color: $main-color;
+    z-index: 1;
+    @include circle($center-size);
+  }
+
+  &__seconds {
+    position: absolute;
+    top: $seconds-padding;
+    width: 3px;
+    border-radius: 10px;
+    height: $seconds-height;
+    background-color: $seconds-background-color;
+    transform-origin: bottom;
+    animation: rotation 60s linear infinite;
```

Comment:
It would be great to create variables for seconds too

Reply:
Ok